demo/lightning/text-based/single-node/model.py

```diff
@@ -134,17 +136,29 @@ def format_data(raw_bytes):
 
 class ParquetIterableDataset(df_iter.DataFluxIterableDataset):
 
-    def __init__(self, columns, batch_size, *args, **kwargs):
+    def __init__(self, columns, batch_size, no_dataflux, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.columns = columns
         self.batch_size = batch_size
         self.current_file = None
         self.columns = columns
+        self.no_dataflux = no_dataflux
         """
         A sublcass of the DatafluxIterableDataset, this dataset allows for the reading and batch
         distribution of a parquet file, where batch-size corresponds to rows of the parquet table.
         """
 
+    def fsspec_iter(self, objects):
+        """
+        Return a generator that reads the given objects using fsspec instead of Dataflux.
+        """
+        for obj in objects:
+            object_name = obj[0]
+            path = f"gs://{self.bucket_name}/{object_name}"
```

Comment:
I'm assuming this just works with background auth?

Reply:
Yes